### PR TITLE
feat : initial commit of a task

### PR DESCRIPTION
**Pull request by rel1c-hub**

### CODE REVIEW

**Review by Eidenn005 — APPROVED** [review]
LGTM

File: src/main/java/mate/academy/bookstore/model/Book.java [hunk 1]
```diff
@@ -39,4 +44,12 @@ public class Book {
...
     @Column(nullable = false)
     private boolean isDeleted = false;
+
+    @ManyToMany
+    @JoinTable(
+            name = "books_categories",
+            joinColumns = @JoinColumn(name = "book_id"),
+            inverseJoinColumns = @JoinColumn(name = "category_id")
+    )
+    private Set<Category> categories = new HashSet<>();
```
Comment (yevheniisynytsia) [inline src/main/java/mate/academy/bookstore/model/Book.java:54]: please exclude from equals and hashcode and to string

File: src/main/java/mate/academy/bookstore/serivce/impl/BookServiceImpl.java [hunk 2]
```diff
@@ -24,20 +25,27 @@ public BookDto save(CreateBookRequestDto requestDto) {
         return bookMapper.toDto(bookRepository.save(book));
```
Comment (yevheniisynytsia) [inline src/main/java/mate/academy/bookstore/serivce/impl/BookServiceImpl.java:24]: please add validation for categories' ids that are passed by user
like 109094203132132 that is not in db

File: src/main/resources/db/changelog/changes/02-add-books.yaml [hunk 3]
```diff
@@ -8,40 +8,70 @@ databaseChangeLog:
             columns:
               - column:
                   name: title
-                  value: "Harry Potter and the Sorcerer's Stone"
+                  value: "Dune"
               - column:
                   name: author
```
Comment (yevheniisynytsia) [inline src/main/resources/db/changelog/changes/02-add-books.yaml:13]: please do not modify the changesets that were merged to main

your colleagues will have checksum fail and will have to fix that problems

**Review by yevheniisynytsia — CHANGES_REQUESTED** [review]
Well done! Left some comments

**Pull request closed without merging**